arches_lingo/serializers.py

```diff
@@ -49,6 +53,35 @@ class Meta:
         root_node = "appellative_status"
         fields = "__all__"
 
+    def validate(self, data):
+        data = super().validate(data)
+        graph_slug = self.Meta.graph_slug
+        PREF_LABEL_LIST_ITEM = ListItem.objects.get(list_item_values__value="prefLabel")
+
+        if data:
+            new_label_language = data["appellative_status_ascribed_name_language"][0]
+            new_label_type = data["appellative_status_ascribed_relation"][0]
+
+            resource_instance_id = self.instance.resourceinstance.pk
+            resource_instance = self.instance.resourceinstance.__class__.as_model(
+                graph_slug, resource_ids=[resource_instance_id]
+            ).get(pk=resource_instance_id)
```

Comment:
I'm looking into adding a developer convenience for this, either a method `.enrich()` and/or overwriting the `.resourceinstance` descriptor on the TileQuerySet so that it's a lazy getter that calls enrich() for you. 👀

Reply:
(Not suggesting to block this PR on it.)

Reply:
@jacobtylerwalls I swear this was working yesterday, but when I try to create a new label for an existing scheme self.instance is `None`. Would you be able to take a look at that in my absence?

Reply:
Done! I think you and I were only testing updates, not creates. I also added a little developer convenience for now.